uw-frame-components/portal/features/partials/features-dialog-template.html

```diff
@@ -0,0 +1,27 @@
+<md-dialog aria-label="new features">
+  <md-toolbar class="md-accent">
+    <div class="md-toolbar-tools">
+      <h1>{{ latestFeature.title }}</h1>
+      <span flex></span>
+      <md-button class="md-icon-button" ng-click="closeDialog('dismissed')" aria-label="close features dialog">
+        <md-icon>close</md-icon>
+      </md-button>
+    </div>
+  </md-toolbar>
+    <md-dialog-content>
+      <div class="md-dialog-content features-popup" layout="column" layout-align="center center">
+        <img ng-src="{{ latestFeature.img }}" aria-describedby="featureContent" hide-xs>
+        <p id="featureContent" class="sr-only">{{ latestFeature.description }}</p>
+        <p hide-gt-xs>{{ latestFeature.description }}</p>
```

Comment:
At `xs` is a screen reader going to try to read `latestFeature.description` twice?